src/main/java/com/health/model/QueueManagement.java

```diff
@@ -0,0 +1,510 @@
+
+                }
+                api_url = documentSb.toString();
+                logger.info("API_URL:{}", api_url);
+
+                request = new HttpPost(api_url);
+
+            }
+
+            else if (getRequestType().equals(CommonData.UPDATE_DOCUMENT_RANK)
+                    || getRequestType().equals(CommonData.DELETE_DOCUMENT)) {
+                if (getRequestType().equals(CommonData.UPDATE_DOCUMENT_RANK)) {
+                    documentSb.append(commonData.elasticSearch_url);
+                    documentSb.append("/");
+                    documentSb.append("updateDocumentRank");
+                    documentSb.append("/");
+                    documentSb.append(documentId);
+                    documentSb.append("/");
+                    documentSb.append(documentType);
+                    documentSb.append("/");
+                    documentSb.append(languageId);
+                    documentSb.append("/");
+
+                    documentSb.append(rank);
+
+                }
+
+                else if (getRequestType().equals(CommonData.DELETE_DOCUMENT)) {
+                    documentSb.append(commonData.elasticSearch_url);
+                    documentSb.append("/");
+                    documentSb.append("deleteDocument");
+                    documentSb.append("/");
+                    documentSb.append(documentId);
+                    documentSb.append("/");
+                    documentSb.append(documentType);
+                    documentSb.append("/");
+                    documentSb.append(languageId);
+
+                }
+
+                api_url = documentSb.toString();
+                logger.info("API_URL:{}", api_url);
+
+                request = new HttpGet(api_url);
+
+            }
+
+            if (request instanceof HttpPost) {
+                List<NameValuePair> paramsforAddDocument = new ArrayList<>();
+                paramsforAddDocument.add(new BasicNameValuePair("documentPath", getDocumentPath()));
+                paramsforAddDocument.add(new BasicNameValuePair("documentUrl", getDocumentUrl()));
+                paramsforAddDocument.add(new BasicNameValuePair("view_url", getViewUrl()));
+                paramsforAddDocument.add(new BasicNameValuePair("categoryId", Integer.toString(getCategoryId())));
+                paramsforAddDocument.add(new BasicNameValuePair("category", getCategory()));
+                paramsforAddDocument.add(new BasicNameValuePair("topicId", Integer.toString(getTopicId())));
+                paramsforAddDocument.add(new BasicNameValuePair("topic", getTopic()));
+                if (getOutlinePath() != null && !getOutlinePath().isEmpty()) {
+                    paramsforAddDocument.add(new BasicNameValuePair("outlinePath", getOutlinePath()));
+
+                }
+                httpPost = (HttpPost) request;
+                httpPost.setEntity(new UrlEncodedFormEntity(paramsforAddDocument, "UTF-8"));
+
+                response = httpClient.execute(httpPost);
```

Comment:
Remove this line.

Put the following line after the if. 

response = httpClient.execute(request);